### DISCUSSION TITLE
rAudio 1 • 20211218: Total time of track doesn't change if track changes

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#709] [Fixed Issues]
In the playback screen the total time of the track doesn't (always) change as the next track starts. In that case you have to refresh the window.

**Comment by rern** [comment]
Total time, elapsed and progress bar which are not good on track changed were fixed in UPDATE. Please try it.

**Reply by klassikmann** [reply]
Looks promising. Thank you.

**Comment by klassikmann** [comment]
rAudio 1 • 20211222: Fixed
